en/components/grid.md

```diff
@@ -543,6 +543,144 @@ public ngOnInit() {
+
+For `string` and `boolean` data type is available the following function:
+ - count
+
+For `number` data type are available the following functions:
+ - count
+ - min
+ - max
+ - average
+ - sum
+
+For `date` data type are available the following functions:
+ - count
+ - earliest
+ - latest
+
+**Grid summaries** are enabled per-column by setting `hasSummary` property to `true`. It is also important to keep in mind that the summaries for each column are resolved accordind column data type. In the `igx-grid` default column data type is `string`, so if you want `number` or `date` specific summaries you should specify `dataType` property to `number` or `date`.
+
+```typescript
+<igx-grid #grid1 [data]="data" [autoGenerate]="false" height="800px" width="800px" (onColumnInit)="initColunm($event)" >
+    <igx-column field="ProductID" header="Product ID" width="200px"  [sortable]="true">
+    </igx-column>
+    <igx-column field="ProductName" header="Product Name" width="200px" [sortable]="true" [hasSummary]="true">
+    </igx-column>
+    <igx-column field="ReorderLevel" width="200px" [editable]="true" [dataType]="'number'" [hasSummary]="true">
+    </igx-column>
+</igx-grid>
+```
+
+The other way to enable summary for specific column or a list of columns is to use the public method `enableSummaries` of the **igx-grid**.
+
+```typescript
+<igx-grid #grid1 [data]="data" [autoGenerate]="false" height="800px" width="800px" (onColumnInit)="initColunm($event)" >
+    <igx-column field="ProductID" header="Product ID" width="200px"  [sortable]="true">
+    </igx-column>
+    <igx-column field="ProductName" header="Product Name" width="200px" [sortable]="true" [hasSummary]="true">
+    </igx-column>
+    <igx-column field="ReorderLevel" width="200px" [editable]="true" [dataType]="'number'" [hasSummary]="false">
+    </igx-column>
+</igx-grid>
+<button (click)="enableSummary()"> Enable Summary</button>
+```
+```typescript
+public enableSummary() {
+    this.grid1.enableSummaries([{fieldName: "ReorderLevel", customSummary: this.mySummary},
+    {fieldName: "ProductID"}]);
+  }
+```
+
+If these functions do not fulfill your requirements you can provide a custom summary for the specific columns. In order to achieve this you have to override one of the base classes `IgxSummaryOperand`, `IgxNumberSummaryOperand`, `IgxDateSummaryOperand` according to the column data type and your needs. In this way you can redifine the existing or you can add new functions. `IgxSummaryOperand` class provide default implementation only for `count` method. `IgxNumberSummaryOperand` extends `IgxSummaryOperand` and provide implementation for methods: `min`, `max`, `sum` and `average`. `IgxDateSummaryOperand` extends `IgxSummaryOperand` and additionally gives you: `earliest` and `latest`.
```

Comment:
one of the base classes `IgxSummaryOperand`, `IgxNumberSummaryOperand`, or `IgxDateSummaryOperand`
In this way you can redefine the existing function or you can add new functions.
The `IgxSummaryOperand` class provides the default implementation
and provides implementation for the `min`, `max`, `sum`, and `average` methods.
additionally gives you `earliest` and `latest` methods.

Reply:
The IgxSummaryOperand class provides the default implementation
and provides implementation for the min, max, sum, and average methods.

... this sentence is not correct, IgxSummaryOperand  does not provides mplementation for the min, max, sum, and average methods.